units/en/unit3/hands-on.mdx

```diff
@@ -0,0 +1,103 @@
+# Hands-on
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1QNfJNqJ6_D9nnf7QeGEW9R3YvWQRxIKY?usp=sharing)
```

Comment:
As mentioned in the previous PR, it's better to host the notebooks somewhere public (e.g. Hub or GitHub, or even automatically generate from the mdx)